### PR TITLE
Floating Menus: Deduplicate Replace Media 

### LINKED ISSUE #10873: Floating Menus: Deduplicate Replace Media
Replace media is an action on the floating menu and the quick action menu. 

Remove it from the floating menu because background element doesn't have a floating menu but it does have a quick action menu, so this keeps the placement of replace media consistent.

Screen Shot 2022-03-09 at 1 31 55 PM

### PR DESCRIPTION
**Pull request by BrookeGraham**
## Context
Replace media is an action on the floating menu and the quick action menu.

Remove it from the floating menu because background element doesn't have a floating menu but it does have a quick action menu, so this keeps the placement of replace media consistent.

## Testing Instructions

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. check that replace media is removed from floating menus for all media types ie. video. images, etc.

### Does this PR have a security-related impact?
no

### Does this PR change what data or activity we track or use?
no

### Does this PR have a legal-related impact?

no
## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #10873

### CODE REVIEW

File: packages/story-editor/src/components/floatingMenu/menus/image.js [hunk 1]
```diff
@@ -36,8 +35,6 @@ import {
 const FloatingImageMenu = memo(function FloatingImageMenu() {
   return (
     <>
-      <SwapMedia />
-
       <Separator />
```
Comment (BrittanyIRL) [inline packages/story-editor/src/components/floatingMenu/menus/image.js]: let's remove the separator here (as well as on video menu)
Screen Shot 2022-03-09 at 5 13 05 PM

**Review by samwhale — APPROVED** [review]
Once the separators are removed it's good for me 👍

File: packages/story-editor/src/components/floatingMenu/menus/image.js (hunk 1 above)
Comment (BrookeGraham) [reply packages/story-editor/src/components/floatingMenu/menus/image.js]: 🤦🏻‍♀️

**Comment by BrookeGraham** [comment]
Thanks Brittany, fixed.

**Review by BrittanyIRL — APPROVED** [review]

**Pull request merged**